svc-bgs-api/src/docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+
+# TODO: get it working with bitnami/ruby image
+# `bundle install` failed with bitnami/ruby:2.7.6
+
+# Or get alpine image working
+# FROM ruby:2.7.6-alpine
+# RUN apk update && apk --no-cache add expat=2.5.0-r0 git && rm -rf /var/cache/apk/*
```

Comment:
@greene858 You can play with the base image to minimize SecRel alerts. I ran into issues when trying `bitnami/ruby:2.7.6` and `ruby:2.7.6-alpine`.

Reply:
I'm using `2.7.6` due to Yang's branch: https://github.com/department-of-veterans-affairs/abd-vro/blob/nanotone/rails-dev-image/svc-bgs-api/Dockerfile#L1